src/postprocess/band_structure.jl

```diff
@@ -162,6 +175,17 @@ function detexify_kpoint(string)
     string
 end
 
+function retexify_kpoint(string)
```

Comment:
Not sure about this. Probably we can dump both `retexify_kpoint` and `detexify_kpoint` now that pymatgen is out of the loop.

Reply:
Agree, can't we just remove all this?